core/server/data/migrations/versions/3.18/001-add-members_stripe_connect-auth-permissions.js

```diff
@@ -0,0 +1,56 @@
+const ObjectId = require('bson-objectid');
+const {logging} = require('../../../../lib/common');
```

Comment:
logging is now in `core/shared/logging` :eyes:

Reply:
Fixed in latest commit [https://github.com/TryGhost/Ghost/pull/11861/commits/a13c471b572735b1755715f3925dab4c365a877e ](https://github.com/TryGhost/Ghost/pull/11861/commits/a13c471b572735b1755715f3925dab4c365a877e)